app/types/transport/requests/teammate/create-teammate.ts

```diff
@@ -16,7 +21,7 @@ export interface CreateTeammateMessagePayload {
    * Contact of creating teammate
    */
   contacts?: [ {
-    type: ContactType,
+    type?: ContactType,
```

Comment:
this leads to compilation error

![image](https://user-images.githubusercontent.com/31101125/155021579-758b49dd-367d-4620-bf15-5c4c958c8656.png)